lib/datadog/core/utils/backport.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module Datadog
+  module Core
+    module Utils
+      # Methods from future versions of Ruby implemented in for older rubies.
+      #
+      # This helps keep the project using newer APIs for never rubies and
+      # facilitates cleaning up when support for an older versions of Ruby is removed.
+      module Backport
+        # `String` class backports.
+        module String
```

Comment:
Minor: As more of a "futurology" comment (e.g. thinking of how this may perhaps evolve in the future), maybe it would make sense to have a more top-level module to contain a bag of backports (e.g. `Datadog::Core::Backport`) or even per-minimum-Ruby-version-to-avoid-the-backport (`Datadog::Core::BackportFrom24`) versus having it scoped into `Utils` > `String` per-class.

Reply:
I'm not sure I understand the suggestion, could you expand the examples a bit more?